tests/sentry/api/endpoints/test_broadcast_index.py

```diff
@@ -60,6 +62,24 @@ def test_basic_user_with_all(self):
         assert len(response.data) == 1
         assert response.data[0]["id"] == str(broadcast1.id)
 
+    def test_organization_filtering(self):
+        org = self.create_organization()
+        self.create_member(organization=org, user=self.user, role="admin")
+
+        broadcast1 = Broadcast.objects.create(message="foo", is_active=True)
+        broadcast2 = Broadcast.objects.create(message="bar", is_active=True)
```

Comment:
This is more like cleanup, but could you move the broadcast creation into a `setUp`? I noticed we use them in every test in this testcase